pkg/services/featuremgmt/registry.go

```diff
@@ -15,8 +15,13 @@ var (
 		},
 		{
 			Name:        "envelopeEncryption",
-			Description: "encrypt secrets",
-			State:       FeatureStateBeta,
+			Description: "Enable envelope encryption",
+			State:       FeatureStateDeprecated,
```

Comment:
Does it makes sense to keep it as deprecated? If I'm not wrong it shouldn't be used anywhere in the code, so I also thought about directly removing it (basically replacing `envelopeEncryption` by `disableEnvelopeEncryption`) but I was unsure. Any thoughts?